.github/workflows/sdk_publish.yaml

```diff
@@ -0,0 +1,21 @@
+name: Publish
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (blocking):** Should we use this publish workflow or existing changesets workflow?

If we get a specific benefit with this publish workflow, we probably need to add this new package to the list of ignored packages in the [changesets config](https://github.com/vercel/vercel/blob/40b7ee0d297c212961279639d9c73d4fed2312f8/.changeset/config.json#L13).

Reply:
Sounds worth using, then. I [just added](https://github.com/vercel/vercel/pull/12039/commits/501e692bec8f814d2e567bb2e5a4920e4c992555) the `@vercel/sdk` package to the list of ignored packages for changesets. I think that'll do what we want here.

Reply:
Turns out that changesets doesn't like ignoring a thing it can't find. We can add that ignore later.